src/CodeGenerator/InjectorGenerator.php

```diff
@@ -98,6 +98,8 @@ private function buildFromTemplate(string $templateFile, string $outputFile, arr
 
     private function generateInjector(): void
     {
+        assert(is_string($this->outputDirectory));
```

Comment:
Imho, thats not safe. This is only true in case someone called the setter. Using php assert in this case is not how it should be used.

https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2021-08-02-TSC-Minutes.md#allow-php-assert-along-with-webmozart-assert-in-specific-cases